counter/src/main/resources/schema/infinispan-counters-config-13.0.xsd

```diff
@@ -12,15 +12,22 @@
                 <xs:element name="strong-counter" type="tns:strong-counter">
                     <xs:annotation>
                         <xs:documentation>
-                            Configures a strong consistent counter. Its state is well know before and after an update.
+                            Configures strong consistent counters.
+                            The value of a strong counter is stored in a single key for consistency.
+                            During updates to the counter, the value is known. Updates to the counter value are performed under the key lock.
```

Comment:
This is the kind of information that won't want to exposed to users.
It be false when we stop using a cache and start using JGroups (WIP https://github.com/pruivo/JGroups/tree/t_counters_changes and https://github.com/pruivo/JGroups/tree/t_pn_counter)